data/json/items/resources/metal.json

```diff
@@ -77,8 +77,8 @@
     "category": "spare_parts",
     "name": { "str_sp": "silver" },
     "description": "A soft shiny metal.  Before the Cataclysm it was worth quite a bit but its value is now greatly diminished.",
-    "weight": "3 g",
-    "//": "Density 10.40g/cm³ ~ 2.6kg/250ml @ stack 1000 = 3g/unit",
+    "weight": "2623 mg",
+    "//": "Density 10.40g/cm³ ~ 524.5g/50ml @ stack 200 = 2623mg/unit",
```

Comment:
`10.40g/cm³` is a typo and should be 10.49. This is fixed in a later commit. `524.5g` is correct.